tests/test_gpu_common.py

```diff
@@ -706,3 +706,32 @@ def test_xcor_from_saved(self, gpu_fit):
         op.apply(time_M=nt-1)
 
         assert np.all(g.data == 30)
+
+
+class TestDeviceScheduling(object):
+
+    def get_deviceid(self, op):
```

Comment:
Maybe this method would be more reusable if it was placed in the Operator class

Reply:
but I don't see why we should ever need it aside from testing

the better way, btw, would be adding it to the SymbolRegistry next to NThreads etc